about.html

```diff
@@ -75,13 +75,13 @@ <h2>Name</h2>
             </div>
             
         </div>
-        <div class="person5">
-            <img src="\img\aboutImg\dummyImg.jpg" alt="Team member picture">
+        <div class="person Karen">
+            <img src="https://avatars1.githubusercontent.com/u/47998177?s=400&v=4" alt="Karen's picture">
             <h2>Name</h2>
```

Comment:
you are probably ready to start adding people's names to the h2 sections so we see the effect of the content